cg/meta/workflow/balsamic.py

```diff
@@ -436,55 +436,39 @@ def get_normal_sample_name(self, case_id: str) -> Optional[str]:
 
     def get_latest_observations(self, observations_pattern: str) -> Optional[str]:
         """
-        Returns the latest loqusdb dump file (loqusdb_<observations>_export-<date>-.vcf.gz) matching an
+        Returns the latest Loqusdb dump file (loqusdb_<observations>_export-<date>-.vcf.gz) matching an
         observations pattern.
         """
+        available_files: iter = sorted(
+            Path(self.loqusdb_path).glob(f"*{observations_pattern}*"),
+            key=lambda file: file.stem.split("-"),
+            reverse=True,
+        )
 
-        available_files: iter = Path(self.loqusdb_path).glob(f"*{observations_pattern}*")
-        return get_last_element_from_iterator(available_files)
+        return str(available_files[0]) if available_files else None
+
+    def get_verified_observations(self, observations: List[str]) -> dict:
+        """Returns a verified {option: path} observations dictionary."""
+        observations_dict = dict()
+        for wildcard in list(ObservationFileWildcards):
+            path: str = get_string_from_list_by_pattern(observations, wildcard)
```

Comment:
```suggestion
            file_path: str = get_string_from_list_by_pattern(observations, wildcard)
```
Not really a path is it (judging from the type hint).